Pod/Classes/VOKCoreDataManager.m

```diff
@@ -631,36 +682,38 @@ - (void)saveAndMergeWithMainContextAndWait:(NSManagedObjectContext *)context;
 
 #pragma mark - Background Importing
 
-+ (void)writeToTemporaryContext:(VOKWriteBlock)writeBlock
+- (void)writeToTemporaryContext:(VOKWriteBlock)writeBlock
```

Comment:
Same thing in a couple places here, regarding that compiler macro for replaced methods (if we find it)